### PR TITLE
Show counts in admin panel and fix error handling

### LINKED ISSUE #76: Should be able to see how many of the active users have voted on an election

### PR DESCRIPTION
**Pull request by ekmartin**
* Shows total number of votes and total number of active users in the edit election frontend (fixes #76).
* Fixes preloading of elections in the election API (`app.param`).
* Makes use of express middleware to handle errors
* Adds 404 middleware for the API

### CODE REVIEW

File: features/admin.feature [hunk 1]
```diff
@@ -73,3 +73,8 @@ Feature: Admin
     When I create a user
     Then I see alert "Bruker registrert!"
 
+  Scenario: Count number of votes and active users
+    Given There is an active election
+    And The election has votes
+    And I am on the edit election page
+    Then I should see number of votes and active users
```
Comment (relekang) [inline features/admin.feature]: Wouldn't it be better to be explicit than to have "magick" numbers in the step definition?

```gherkin
Then I should see 1 in voted count
And I should see 2 in active user count
```

**Comment by relekang** [comment]
Screenshot? :)

File: features/admin.feature (hunk 1 above)
Comment (ekmartin) [inline features/admin.feature]: Smart, changed it now :)

**Comment by ekmartin** [comment]
image

**Comment by ekmartin** [comment]
image

**Comment by relekang** [comment]
LGTM :smile:

**Comment by JuulArthur** [comment]
Looks good!

**Pull request merged**